service/src/main/java/bio/terra/workspace/service/resource/controlled/ControlledResourceService.java

```diff
@@ -560,16 +562,19 @@ private JobBuilder commonDeletionJobBuilder(
         .newJob()
         .description(jobDescription)
         .jobId(jobId)
-        .flightClass(DeleteControlledResourceFlight.class)
+        .flightClass(DeleteControlledResourcesFlight.class)
```

Comment:
haha i'm trying to do that but it causes some json serialization problem :(

Reply:
If you're not able to do this, can you add comments to DeleteControlledResourcesFlight? Eg

This flight can either be used to delete a single resource, or multiple resources.

Single resource deletion uses flight parameters X Y Z and happens in XX.

 Multiple resource deletion uses flight parameters A B C and happens in YY.

Reply:
ah. json cannot serialize a generic list. Changing to an arraylist works!

the resourceType, resourceName and stewardship needs to stay because UI is currently using it for filtering flight job (to track if a resource is deleted).

Jenn says that it's oK to not track the individual resource deletion progress for folder deletion but i don't want to remove the filtering yet for single resource deletion.